README.md

```diff
@@ -427,15 +426,60 @@ RULE:     - the owning team will provide timely, high quality updates for the
+RULE:       dependency for e.g. a CVE fix will be moved to a new major version.
+RULE:       Being vendored that does gladly at least not imply incompatibility
+RULE:       issues with other packages or the SRU policy. But it might happen
+RULE:       that this triggers either:
+RULE:       a) The need to adapt the current version of the main package and/or
+RULE:          other vendored dependencies to work with the new dependency
+RULE:       b) The need to backport the fix in the dependency as the main
+RULE:          package will functionally only work well with the older version
+RULE:       c) The need to backport the fix in the dependency, as it would imply
+RULE:          requiring a newer toolchain to be buildable that isn't available
+RULE:          in the target release.
+RULE: - The rust ecosystem currently isn't yet considered stable enough for
+RULE:   classic lib dependencies and transitions in main; therefore the
+RULE:   expectation for those packages is to vendor (and own/test) all
+RULE:   dependencies (except those provided by the rust runtime itself).
+RULE:   This implies that all the rules for vendored builds always
+RULE:   apply to them. In addition:
+RULE:   - The rules and checks for rust based packages are preliminary and might
+RULE:     change over time as the ecosytem matures and while
+RULE:     processing the first few rust based packages.
+RULE:   - It is expected rust builds will use dh-cargo so that a later switch
+RULE:     to non vendored dependencies isn't too complex (e.g. it is likely
+RULE:     that over time more common libs shall become stable and then archive
+RULE:     packages will be used to build).
+RULE:   - the rust tooling e.g. dh-cargo can not yet automatically provide
+RULE:     all that we require - for example Cargo.lock - until it does the
+RULE:     package build shall be adapted in a way to provide that without
+RULE:     dh* tooling support.
+RULE: - All vendored dependencies (no matter what language) shall have a
+RULE:   way to be refreshed
 TODO-A: - This does not use static builds
 TODO-B: - The team TBD is aware of the implications by a static build and
 TODO-B:   commits to test no-change-rebuilds and to fix any issues found for the
 TODO-B:   lifetime of the release (including ESM)
+
 TODO-A: - This does not use vendored code
 TODO-B: - The team TBD is aware of the implications of vendored code and (as
-TODO-B:   alerted by the security team) commits to provide updates to the security
-TODO-B:   team for any affected vendored code for the lifetime of the release
-TODO-B:   (including ESM).
+TODO-B:   alerted by the security team) commits to provide updates and backports
+TODO-B:   to the security team for any affected vendored code for the lifetime
+TODO-B:   of the release (including ESM).
+
+TODO-A: - This does not use vendored code
+TODO-B: - This package uses vendored go code tracked in go.mod as shiped in the
```

Comment:
the direct dependencies are in `go.mod`. Exact checksum content for direct and indirect dependencies are in `go.sum`. Reproducible builds involve having a `go.sum` file.

Reply:
Thanks, I adopted the section (and one other reference to it) marking resolved